packages/app/components/strategies/StrategyCard.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { cx } from "class-variance-authority";
+
+import { Button, CaptionText, Icon } from "@/ui";
+import { Strategy, useStrategyContext } from "@/contexts";
+import { TokenLogoPair } from "@/components/TokenLogoPair";
+
+import { FREQUENCY_LABEL } from "./constants";
+
+interface StrategyCardProps {
+  strategy: Strategy;
+}
+
+export const StrategyCard = ({ strategy }: StrategyCardProps) => {
+  const [isHovered, setIsHovered] = useState(false);
+  const [isSelected, setIsSelected] = useState(false);
+  const { selectedStrategy, setSelectedStrategy } = useStrategyContext();
+
+  const { buyToken, sellToken } = strategy;
+
+  const strategyTotalDetails = [
+    {
+      label: "Total Amount",
+      totalAmount: `${strategy.totalSellAmount} ${sellToken.symbol}`,
+    },
+    { label: "No. of days", totalAmount: strategy.daysAmount },
+  ];
+
+  useEffect(() => {
+    setIsSelected(selectedStrategy?.id === strategy.id);
+  }, [selectedStrategy, strategy.id]);
+
+  return (
+    <div
+      className={cx(
+        "p-3 bg-white shadow-md min-w-60 rounded-2xl cursor-pointer",
+        {
+          "!bg-primary-900": isSelected,
+          "shadow-lg": isHovered,
```

Comment:
can this be done with css only? `hover:shadow-lg`

Reply:
Yes

Reply:
IMO the hover shadow effect isn't working, it's not noticeable. I would discuss with @akashtyagi027 adding a subtle change to the bg.

I think `hover:bg-surface-25` works much better.

Reply:
We need to change 2 different styles on hover:
* The card shadow 
* The "select" button style

The `hover:` from Tailwind is the pseudo-selector for a specific element. If we want to target the two elements (the card + the button inside it) we need a way to achieve this other than the pseudo-selector :c

Agree that maybe instead of using `lg` shadow on hover we could use another styling to be more impactful. Will send the link to this convo to Akash c:

Reply:
I don't think we need to highlight the button when hovering the card. We only need to highlight the card with a different box shadow.

Reply:
Not sure if it's necessary, but It is also possible to change the `bg` on the `button` when hovering the card in `tailwind` with `group-hover` selector.

docs:
https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups

Reply:
Whaaaaaaaaaaaat that's so cool hahaha didn't knew about these pseudo-modifies, good one Diogo! Thanks a lot :D

I'll get this into the design chat and check with Akash c: